config/locales/en.yml

```diff
@@ -20,4 +20,6 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  errors:
```

Comment:
We can remove this now!  In my digging, I realized that carrier wave has a separate en.yml file, so this was actually not where we needed to set the translation.  But the carrier wave gem update will be sufficient.

Reply:
Oh wait, you're totally right, this is getting loaded!  Nice work!  This can stay!

Reply:
I think we went through the exact same thought process on this, from: "we can remove this!" to "actually, it works, let's keep it!" 😄